### PR TITLE
fix doc error

### PR DESCRIPTION
**Pull request by firedpeanut**
path error

### CODE REVIEW

File: examples/crowd-funding/README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ export LINERA_STORAGE1="rocksdb:$(dirname "$LINERA_WALLET1")/linera.db"
 export LINERA_WALLET2="$PWD/target/debug/wallet_2.json"
 export LINERA_STORAGE2="rocksdb:$(dirname "$LINERA_WALLET2")/linera_2.db"
 
-(cargo build && cd examples && cargo build --release)
+(cargo build && cd examples && cargo build --release && cd ..)
```
Comment (afck) [inline examples/crowd-funding/README.md:57]: Good catch, thank you!

But the README.md is auto-generated from the lib.rs, so the error needs to be fixed there (too).

Comment (firedpeanut) [reply examples/crowd-funding/README.md:57]: I see

**Review by afck — APPROVED** [review]
Thanks! :+1:

File: examples/crowd-funding/README.md (hunk 1 above)
Comment (ma2bd) [reply examples/crowd-funding/README.md:57]: @afck ´cd ..´ does nothing here

Comment (ma2bd) [reply examples/crowd-funding/README.md:57]: I would suggest to revert the change

Comment (afck) [reply examples/crowd-funding/README.md:57]: Ah, because of the brackets! (It's all in a subshell.)
I can revert it in my next PR. Sorry!

Comment (firedpeanut) [reply examples/crowd-funding/README.md:57]: sorry. yes subshell

**Pull request merged**